api/holodeck/v1alpha1/types.go

```diff
@@ -216,3 +218,13 @@ type NVIDIAContainerToolkit struct {
 	// +optional
 	EnableCDI bool `json:"enableCDI"`
 }
+
+// Kernel defines the kernel configuration
+type Kernel struct {
+	// Modify specifies whether to modify the kernel version
+	Modify bool `json:"modify"`
```

Comment:
Is there a use case to install the latest and greatest kernel version? I would prefer having one string field called `KernelVersion` instead

Reply:
I will remove the `Modify`, leaving only the `Version` that way we have `Kernel.Version`